packages/next/src/client/components/layout-router.tsx

```diff
@@ -302,6 +302,9 @@ function InnerLayoutRouter({
   childProp,
   segmentPath,
   tree,
+  // `buildId` prop is for adding buildId into RSC payload for caching purpose.
+  // Then same page will be unique among different builds.
+  buildId: _buildId,
```

Comment:
Can you pass this through context from AppRouter instead? Otherwise it'll increase the payload quite a bit with duplicate identifiers.

Reply:
You can add it to GlobalLayoutRouterContext